### PR TITLE
Javascript functions practice - James Aguirre

### PR DESCRIPTION
**Pull request by james-aguirre**
Screenshot 2023-03-21 at 7 56 42 PM

### CODE REVIEW

**Comment by james-aguirre** [comment]
Screenshot 2023-03-21 at 7 45 38 PM
Screenshot 2023-03-21 at 7 46 38 PM

File: javascript-functions-practice/get-area-of-circle.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/* exported getAreaOfCircle */
+function getAreaOfCircle(radius) {
+  return (radius * radius) * 3.141592653589793;
```
Comment (shawnkost) [inline javascript-functions-practice/get-area-of-circle.js:3]: There's also a method you can use here, so you don't need to manually type it out, `Math.PI`. What you did isn't wrong, I just wanted to show you another way!

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/PI

**Review by shawnkost — APPROVED** [review]
Nicely done James! 👍🏻

File: javascript-functions-practice/get-area-of-circle.js (hunk 1 above)
Comment (james-aguirre) [reply javascript-functions-practice/get-area-of-circle.js:3]: @shawnkost That's good to know, thanks!

**Pull request merged**